pkg/snapstore/snapstore_test.go

```diff
@@ -459,6 +460,55 @@ var _ = Describe("Dynamic access credential rotation test for each provider", fu
 	}
 })
 
+var _ = Describe("Blob Service URL construction for Azure", func() {
+	var credentials *azblob.SharedKeyCredential
+	BeforeEach(func() {
+		var err error
+		// test strings
+		storageAccount, storageKey := "testAccountName", "dGVzdEFjY291bnRLZXk="
+		credentials, err = azblob.NewSharedKeyCredential(storageAccount, storageKey)
+		Expect(err).ShouldNot(HaveOccurred())
+	})
+	Context("when the environment variable \"EMULATOR_ENABLED\" is not set", func() {
+		It("should return the default blob service URL", func() {
+			blobServiceURL, err := ConstructBlobServiceURL(credentials)
+			Expect(err).ShouldNot(HaveOccurred())
+			Expect(blobServiceURL.String()).Should(Equal(fmt.Sprintf("https://%s.%s", credentials.AccountName(), brtypes.AzureBlobStorageHostName)))
+		})
+	})
+	Context("when the environment variable \"EMULATOR_ENABLED\" is set", func() {
```

Comment:
Use the constant `envEmulatorEnabled` instead of `EMULATOR_ENABLED`

Reply:
Addressed in [6fca638](https://github.com/gardener/etcd-backup-restore/pull/699/commits/6fca638920bac61b1739494207b0f965596836c7).